File: my-signalr-chathub_backend/my-signalr-chathub_backend/DTOs/SuperTienda/OrderDetailDto.cs
public class OrderDetailDto
{
    public double LineaDetalle { get; set; }
    public string IdPedido { get; set; }
    public string ArticuloId { get; set; }
    public double? Cantidad { get; set; }
    public double? Descuento { get; set; }
    public double? CosteEnvío { get; set; }
}

File: my-signalr-chathub/src/dtos/supertienda/orderDetailDto.ts
export interface orderDetailDto {
    lineaDetalle: number;
    idPedido: string;
    articuloId: string;
    cantidad?: number;   // Optional 'number' type for nullable double
    descuento?: number;  // Optional 'number' type for nullable double
    costeEnvio?: number; // Optional 'number' type for nullable double
  }
  